packages/drivers/routerlicious-driver/src/restWrapperBase.ts

```diff
@@ -0,0 +1,121 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { AxiosRequestConfig, AxiosRequestHeaders } from "axios";
+
+export abstract class RestWrapper {
```

Comment:
Do we have to clone it? It's probably fine if we are trying to get rid of any server dependency. If not, feels like no changes are required here, no?

Reply:
This doesn't remove the server dependency because the server dependency for RestLessClient is still needed.

@jatgarg I know there is an initiative to reduce client-server dependency, but I do not feel that the R11sDriver is a priority for that. If it is a priority, I'd prefer that we be be more intentional about it.

Reply:
I made some changes in this file too. However based on your other comments we might not need them but still I want to keep this in client repo as this file seems prone to changes now or in future.

Reply:
@znewton Yes r11s is a priority as it is used in AFR. We want to improv perf/reliability here in near future and that is why we are moving towards breaking the dep. Also, this PR is not moving everything over to client but I am opening a bug to do so. For now, I mostly moved what is required.

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/4077